src/OpalCompiler-Core/OCInstanceScope.class.st

```diff
@@ -30,8 +30,7 @@ OCInstanceScope >> hasBindingThatBeginsWith: aString [
 OCInstanceScope >> initialize [
   
 	vars := Dictionary new.
-	reservedVars  := (ReservedVariable subclasses collect: [ :class |
-			class instance name -> class instance]) asDictionary 
+	reservedVars  := ReservedVariable lookupDictionary
```

Comment:
I think we can prefill #vars with ReservedVariables and remove #reservedVars at all